pkg/webhook/service/service.go

```diff
@@ -25,26 +24,14 @@ import (
 	crclient "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-func SetupWebhookCerts(client crclient.Client, ctx context.Context, namespace string) error {
-	log.Info("Attempting to create the secure service")
-	err := createSecureService(client, ctx, namespace)
-	if err != nil {
-		log.Info("Failed creating the secure service")
-		return err
-	}
-	return nil
-}
-
-func createSecureService(client crclient.Client, ctx context.Context, namespace string) error {
+func CreateSecureService(client crclient.Client, ctx context.Context, namespace string, annotations map[string]string) error {
```

Comment:
That's not really create but make sure it exists in actual state... Create or update...